netutils/telnetd/telnetd_daemon.c

```diff
@@ -316,6 +307,33 @@ static int telnetd_daemon(int argc, FAR char *argv[])
           close(drvrfd);
         }
 
+#ifdef CONFIG_NETUTILS_TELNETD_USE_POSIX_SPAWNP
```

Comment:
don't need spawn, we can reuse the caller thread(telnetd main thread) as the context.